src/applications/disability-benefits/526EZ/tests/components/ConfirmationPoll.unit.spec.jsx

```diff
@@ -85,7 +85,7 @@ describe('ConfirmationPoll', () => {
 
   it('should render the confirmation page', done => {
     mockApiRequest(successResponse.response);
-    const tree = shallow(<ConfirmationPoll {...defaultProps} pollRate={10} />);
```

Comment:
This was resulting in `Cannot read property '_currentElement' of null`, using `mount` instead resolved the issue.